guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -39,14 +39,22 @@ On the Discovered Hosts page, power off and then delete the discovered hosts.
 From the *Select an Organization* menu, select each organization in turn and repeat the process to power off and delete the discovered hosts.
 Make a note to reboot these hosts when the upgrade is complete.
 ifdef::satellite[]
-. Ensure that the {Project} Maintenance repository is enabled:
+. Ensure that the {Project} Maintenance repository and the {Project} {ProjectVersion} repository are enabled:
 +
 [options="nowrap" subs="attributes"]
 ----
 # subscription-manager repos --enable \
 {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+
+# subscription-manager repos --enable \
+{RepoRHEL8ServerSatelliteServerProductVersion}
 ----
 
+[NOTE]
+====
+If you find further issues on upgrading the {ProjectServer}, please refer https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html/release_notes/assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues] in the Release Notes as a workaround.
```

Comment:
Should the `{ReleaseNotesURL}` macro be used instead?

Reply:
`{BaseFilenameURL}` is not catching `assembly_introducing-red-hat-satellite_sat6-release-notes`. Hence, we are landing at - https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html/release_notes/index#ref_known-issues_assembly_introducing-red-hat-satellite

instead of 

https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html/release_notes/assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite

Reply:
I don't know why the release notes uses the multi-page view instead of the single page view. Otherwise you could easily link to https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html-single/release_notes/index#ref_known-issues_assembly_introducing-red-hat-satellite.

Another option is to drop the `index#` from the `ReleaseNotesURL` so you can actually link to individual chapters.

@Lennonka @adamlazik1 any thoughts on this?

Reply:
Agree with the @adamlazik1's DDF point. Let's see if the `attributes` route is possible.

Reply:
Oopsie. So, the thing is that the single-page URLs and multi-page URLs are composed differently (not only for RNs, but for all guides) -- the URL paths have these formats:

- single-page `/index#<module_id>`
- multi-page `/<assembly_id>#<module_id>`

That means `{MultiBaseURL}` cannot be used with `index#`, because it must point to the ID of the assembly under which the module is nested. This would make the conversion to multi-page targets more too complicated than I expected.

Perhaps we should stick with single-page links after all. (This would have to be fixed in `attributes-satellite.adoc` as well.)

Regading DDF, we are supposed to have a section on top of each guide on how to use DDF, which should tell the user to switch to multi-page HTML. However, it doesn't generate in Pantheon, probably because the condition is non-applicable for downstream. Outch.

Reply:
@Lennonka Thanks for the clarification!

> In the short term I think this is equivalent, but at least avoids repeating things we already have:

{ReleaseNotesURL} can still be used (as a short term fix) like this:
```
{ReleaseNotesURL}assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues]
```

Reply:
> @Lennonka Thanks for the clarification!
> 
> > In the short term I think this is equivalent, but at least avoids repeating things we already have:
> 
> {ReleaseNotesURL} can still be used (as a short term fix) like this:
> 
> ```
> {ReleaseNotesURL}assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues]
> ```

This seems to be a workaround. Shall I proceed further @Lennonka?

Reply:
I'd go with `{MultiBaseURL}` for now, as Ewoud suggests above.

Reply:
Addressed :+1: